packages/e2e-test-app/jest.setup.js

```diff
@@ -15,7 +15,9 @@ const screenshotDir = './errorShots';
 fs.mkdirSync(screenshotDir, {recursive: true});
 
 // Register to screenshot on each test failure
-global.jasmine.getEnv().addReporter({
+// TODO - use a jest reporter to create screenshots
```

Comment:
Is this the API that wasn't working for us following the Jest 29 upgrade?

Reply:
yes